srcpkgs/python3-fpylll/patches/cython3-legacy.patch

```diff
@@ -0,0 +1,37 @@
+commit b6e12c2b0648e84b26dcf0aac507a5b4d9dde301
+Author: Gonzalo Tornaría <tornaria@cmat.edu.uy>
+Date:   Wed Jul 19 20:38:01 2023 -0300
+
+    cython3 support using legacy directives
+
+diff --git a/setup.py b/setup.py
+index 274836f..8fc5af5 100755
+--- a/setup.py
++++ b/setup.py
+@@ -123,7 +123,12 @@ class build_ext(_build_ext, object):
+             self.extensions,
+             include_path=["src"],
+             build_dir=self.cythonize_dir,
+-            compiler_directives={"binding": True, "embedsignature": True, "language_level": 2},
++            compiler_directives={
++                "binding": True,
++                "embedsignature": True,
++                "language_level": 2
```

Comment:
```suggestion
+                "language_level": 2,
```
My bad, that's what I get for hand-editing the patch after checking... This should fix it.